x-pack/plugins/cases/public/components/user_actions/markdown_form.tsx

```diff
@@ -43,36 +28,13 @@ const UserActionMarkdownComponent = forwardRef<
   UserActionMarkdownProps
 >(({ id, content, caseId, isEditable, onChangeEditable, onSaveContent }, ref) => {
   const editorRef = useRef();
-  const initialState = { content };
-  const { form } = useForm<Content>({
-    defaultValue: initialState,
-    options: { stripEmptyFields: false },
-    schema,
-  });
-
   const fieldName = 'content';
-  const { appId } = useCasesContext();
-  const draftStorageKey = getMarkdownEditorStorageKey(appId, caseId, id);
-  const { setFieldValue, submit } = form;
-
-  const handleCancelAction = useCallback(() => {
-    onChangeEditable(id);
-    removeItemFromSessionStorage(draftStorageKey);
-  }, [id, onChangeEditable, draftStorageKey]);
-
-  const handleSaveAction = useCallback(async () => {
-    const { isValid, data } = await submit();
 
-    if (isValid && data.content !== content) {
-      onSaveContent(data.content);
-    }
-    onChangeEditable(id);
-    removeItemFromSessionStorage(draftStorageKey);
-  }, [content, id, onChangeEditable, onSaveContent, submit, draftStorageKey]);
+  const [fieldValue, setFieldValue] = useState({ content });
 
   const setComment = useCallback(
     (newComment) => {
-      setFieldValue(fieldName, newComment);
+      setFieldValue({ content: newComment });
```

Comment:
Updating the local state when setComment is called through `useImperativehandle`.